### PR TITLE
Docs: README.md

### PR DESCRIPTION
**Pull request by diegotony**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ## Considerations
 
-The functions run in **Serverless Framework** and use **Windows** like OS.
```
Comment (scastillo-jp) [inline README.md:9]: now we don't have functions in windows?

File: README.md [hunk 2]
```diff
@@ -22,24 +22,63 @@ Check Install
 serverless -v
 ```
 
-Install azure plugin
+Install azure plugins
```
Comment (scastillo-jp) [inline README.md:25]: Is necessary a specific version of node to install this plugins?

File: README.md (hunk 1 above)
Comment (diegotony) [reply README.md:9]: No, I found a solution for it, I am trying  to standardize the Function OS

Comment (diegotony) [reply README.md:9]: Only Linux

File: README.md (hunk 2 above)
Comment (diegotony) [reply README.md:25]: No, Only Nodejs should be 10, is due to  azure cli compatibility

**Review by scastillo-jp — APPROVED** [review]

**Pull request merged**